requestid/requestid.go

```diff
@@ -0,0 +1,36 @@
+package requestid
+
+import (
+	"context"
+	"github.com/google/uuid"
```

Comment:
I do not see `Godep` files updated.

Reply:
https://github.com/infobloxopen/atlas-app-toolkit/blob/master/Gopkg.lock#L38